README.md

```
@@ -1,90 +1,79 @@
+## 🔒 What's Enforced
 
-To build the library use:
+**ONE package.json rule:**
+- ✅ `/workspace/package.json` - All dependencies here
+- ❌ `packages/*/package.json` - Blocked by validator
 
-```sh
-npx nx build pkg1
-```
+**Blocked imports:**
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)